internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -899,6 +900,30 @@ public void broadcastTopicSwitch(
     producer.flush();
   }
 
+  /**
+   * Broadcast control message to real-time topic partition, to be consumed by venice leader. Partition high watermarks
+   * are left to local venice leader to prepare and then been produced to version topic partition.
+   *
+   * @param oldServingVersionTopic the version topic change capture consumer should switch from.
+   * @param newServingVersionTopic the version topic change capture consumer should switch to.
+   * @param debugInfo arbitrary key/value pairs of information that will be propagated alongside the control message.
+   */
+  public void broadcastChangeCaptureTopicSwitch(
+      @Nonnull String oldServingVersionTopic,
+      @Nonnull String newServingVersionTopic,
+      Map<String, String> debugInfo) {
+    Validate.notNull(oldServingVersionTopic);
+    Validate.notEmpty(newServingVersionTopic);
```

Comment:
Why do we have different validations for `oldServingVersionTopic` and `newServingVersionTopic`?